io.github.jakubmelka.pdf4qt.json

```diff
@@ -0,0 +1,256 @@
+{
+	"id": "io.github.jakubmelka.pdf4qt",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.15-21.08",
+	"sdk": "org.kde.Sdk",
+	"command": "pdf4qt",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--socket=wayland"
+	],
+	"modules": [{
+			"name": "qbs",
+			"buildsystem": "cmake-ninja",
+			"builddir": true,
+			"config-opts": [
+				"-DWITH_TESTS:BOOL=OFF"
+			],
+			"cleanup": [
+				"*"
+			],
+			"sources": {
+				"type": "archive",
+				"url": "https://download.qt.io/official_releases/qbs/1.23.0/qbs-src-1.23.0.tar.gz",
+				"sha256": "c86aa775446aec728bcbbed782ec128f4e6e2c26536710017343e684bb616d7a"
+			}
+		},
+		{
```

Comment:
Are these generated with one of tools from flatpak-builder-tools? If yes, please move generated dependencies to a separate file; if not, they should be.